### ISSUE TITLE
Save user preferences

### ISSUE THREAD

**Issue by matthew-white** [getodk/central#689]
The user can make a number of selections/choices in Frontend that we've talked about persisting. We've talked about persisting them server-side. Some preferences seem to be top-level/sitewide, while others seem per-project. We already have sitewide settings (for usage reporting), project settings, and form settings. However, those are not specific to the user. This issue is about saving individual users' preferences.

Top-level/sitewide preferences to persist:

- ~Whether the top panel on the homepage is shown or not~ (moved to [#719](https://github.com/getodk/central/issues/719))
- The sort order of the project list

Project preferences:

- ~The sort order of the form list~ (moved to [#719](https://github.com/getodk/central/issues/719))
- Whether the form trash is shown or not - #690
- ~Whether managed or legacy QR codes are shown for app users~ (moved to [#719](https://github.com/getodk/central/issues/719))

**Comment by matthew-white** [comment]
> Once the Frontend PR is merged, and tests are in, we can close this issue.

The Frontend PR is merged. I've filed a separate issue about writing tests: #1049. I think we can go ahead and close this issue.

**Comment by matthew-white** [comment]
@getodk/testers, this issue is now ready to be verified. It is available on the staging server. The main way that this issue is different from #690 is that there is one other user preference to verify:

> The sort order of the project list

Each user's choice/preference for the sort order of the project list should be saved. By default, projects are sorted by recent activity. If I sort projects alphabetically, then I refresh the page or open the page in a new tab, I should continue to see that projects are sorted alphabetically. My selection for the sort order of the project list should not affect what other users see: this preference is limited to my user.

**Comment by srujner** [comment]
@matthew-white To make sure: The sort order of the form list should not be affected by this?

**Comment by matthew-white** [comment]
That's right, at least for now. We've thought about adding another user preference for the sort order of the form list, and that idea is tracked at #719. The main reason we haven't added it is that we've also thought about making it possible to custom-order forms in the form list.
